packages/create-remix/templates/arc/package.json

```diff
@@ -8,6 +8,7 @@
     "start": "cross-env NODE_ENV=production arc sandbox"
   },
   "dependencies": {
+    "@architect/architect": "^10.0.5",
```

Comment:
How can we help ensure this is relatively up to date?

Reply:
couldn't we use `^10.x` to signify any version of `v10`? we don't include lock files in the templates so once they're installed they _should_ use the latest version available, no?

Reply:
otherwise, we can keep it `^10.0.5` and wait for PRs to bump the version

Reply:
Yeah we can def safely set to 10.x

Reply:
`^10.0.0` should indeed just work